ipatests/test_integration/test_caless.py

```diff
@@ -1574,10 +1574,9 @@ def test_server_ipa_ca_install(self):
 
         ca_master = tasks.install_ca(self.master)
         assert ca_master.returncode == 0
-        cert_update_master = self.master.run_command(['ipa-certupdate'])
-        assert cert_update_master.returncode == 0
-        cert_update_replica = self.replicas[0].run_command(['ipa-certupdate'])
```

Comment:
AFAIK `ipa-certupdate` would only be run on the machine `ipa-ca-install` was run on, therefore you need to keep the latter certupdate there.

Reply:
@stlaz we don't want to keep the latter `ipa-certupdate`, because the next step is to do a CA replica install, and we want to check that `ipa-ca-install` does the `ipa-certupdate` behaviour as a first step.

Reply:
@frasertweedale ah, of course, I did not scroll down, my bad

Reply:
Cool, thanks to both for watching 👍

Reply:
Maybe a comment explaining why we do not run `ipa-certupdate` would be useful.

Reply:
done...